src/open_dread_rando/files/schema.json

```diff
@@ -251,9 +251,6 @@
         },
         "starting_items": {
             "type": "object",
-            "properties": {
-                "ITEM_LIFE_SHARDS": {"type": "number", "maximum": 3}
```

Comment:
This item id was only specified to have a max because of the energy bug. Since the bug is fixed and no other item is mentioned, the explicit check is not needed.